ttyd/umbrel-app.yml

```diff
@@ -0,0 +1,23 @@
+manifestVersion: 1
+id: ttyd
+category: DEVELOPMENT
+name: ttyd
+version: "1.7.3"
+tagline: Share your terminal over the web
+description: >-
+  ttyd is a simple command-line tool for sharing terminal over the web.
+releaseNotes: >-
+  Sixel support is off by default now, use -t enableSixel=true to enable it.
```

Comment:
For the initial app release we can either remove `releaseNotes` entirely, or else just say:
` - Initial release for the Umbrel App Store`